docs/data/flink_connectors.yml

```diff
@@ -44,11 +44,11 @@ gcp-pubsub:
   compatibility: ["1.16.x", "1.17.x"]
 
 jdbc:
-  name: "Apache Flink JDBC Connector 3.1.1"
-  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-connector-jdbc-3.1.1/flink-connector-jdbc-3.1.1-src.tgz"
-  source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-jdbc-3.1.1/flink-connector-jdbc-3.1.1-src.tgz.asc"
-  source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-jdbc-3.1.1/flink-connector-jdbc-3.1.1-src.tgz.sha512"
-  compatibility: ["1.16.x", "1.17.x"]
+  name: "Apache Flink JDBC Connector 3.1.2"
+  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-connector-jdbc-3.1.2/flink-connector-jdbc-3.1.2-src.tgz"
+  source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-jdbc-3.1.2/flink-connector-jdbc-3.1.2-src.tgz.asc"
+  source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-jdbc-3.1.2/flink-connector-jdbc-3.1.2-src.tgz.sha512"
+  compatibility: ["1.16.x", "1.17.x", "1.18.x"]
```

Comment:
good catch
it seems same history issue appears for artifacts at 
https://repository.apache.org/content/repositories/orgapacheflink-1704/org/apache/flink/flink-connector-jdbc/
where there are artifacts for 1.16...

I wonder if there is a way to remove it from there without generation a new RC?
Otherwise we probably need another RC

Reply:
I think we can simply support `compatibility: ["1.17.x", "1.18.x"]` which means we only need to build a new rc2 without change any code, WDYT?

Reply:
> I wonder if there is a way to remove it from there without generation a new RC?

Maybe the way does not exists, 1.16 related artifacts is one part of the RC1 and maven staging repo,  creating a new RC is  more safe and recommended.

Reply:
@leonardBang sorry for the confusion
I noticed that despite there is 1.17 in pom, there is nightly/weekly tests for 1.16 as well [1] so we can confirm that tests for Flink 1.16 are passing.
Based on history it came from FLINK-31699. I assumed it could be as a result of failing archunit tests for 1.16. @MartijnVisser please correct me if I'm wrong

[1] https://github.com/apache/flink-connector-jdbc/actions/runs/7723924144

Reply:
I think it's inadvertently bumped Flink version: since the connector still works with Flink 1.16 (as proven in the nightly builds), we should have kept this `flink.version` set to `1.16.0`. Only when we have to drop support for a lower Flink version, we should bump this.

Reply:
I added a PR to fix it in pom and ci https://github.com/apache/flink-connector-jdbc/pull/99

Reply:
It is the issue for v3.1  only
for main it was fixed within FLINK-33378 and seems was not backported to v3.1...

Reply:
Actually I mean  here[1] should be `1.16.0`  instead of `1.16.2`, `1.16.0` is the minimum supported flink version.
[1] https://github.com/apache/flink-connector-jdbc/commit/79e7509256753765813d2e2c970506414de3302a#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R50

Reply:
You did right fix in v3.1 branch ^_^
https://github.com/apache/flink-connector-jdbc/pull/99/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R50

Reply:
ok, got it, thanks
created a PR for that https://github.com/apache/flink-connector-jdbc/pull/102